test/tests/functional/pbs_snapshot_unittest.py

```diff
@@ -55,6 +58,24 @@ def setUp(self):
                                             id="ngpus", expect=True,
                                             sudo=True))
 
+        # Check whether pbs_snapshot binary is accessible
+        try:
+            ret = self.du.run_cmd(cmd=["pbs_snapshot", "-h"])
+            if ret['rc'] == 0:
+                self.pbs_snapshot_exists = True
+            else:
+                self.pbs_snapshot_exists = False
```

Comment:
Ah, thanks for catching that, I'd meant to add that to skip the tests which use pbs_snapshot binary, if the binary is not installed. I've changed the code to reflect that now. Also added logs to show how this will work to the ticket. Please review the changes once more. Thanks!